packages/lighthouse-service/.env.example

```diff
@@ -4,6 +4,12 @@ DB_NAME=
 DB_USER=
 DB_PASSWORD=
```

Comment:
Can we use some other env var name to classify pg db and mongodb

Reply:
I did change pg to LH Db ....i didn't changed other one because just so, the all other microservices is using that for there own DB. Is there any suggested names you have